CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.1.0...HEAD)
+
+## [0.1.0](https://github.com/OpenLineage/OpenLineage/releases/tag/0.1.0) - 2021-08-11
+
+OpenLineage is an _Open Standard_ for lineage metadata collection designed to record metadata for a job in execution. The initial public release includes:
+
+* **An inital specification.** The the inital version [`1-0-0`](https://openlineage.io/spec/1-0-0/OpenLineage.json) of the OpenLineage specification defines the core model and facets.
```

Comment:
can we link to the spec markdown instead? https://github.com/OpenLineage/OpenLineage/blob/0.1.0/spec/OpenLineage.md